### DISCUSSION TITLE
Have any benchmark data compared to golang version?

### DISCUSSION THREAD

**Discussion by aierui** [helyim/helyim#90] [Q&A] [answered]
What does this look like under some common workloads?

**Comment by iamazy** [comment, marked as answer]
Simply conducted a straightforward test using the SeaweedFS `weed benchmark` command, the write performance is quite similar, but the read performance falls behind SeaweedFS.

You can see it here: [benchmark](https://github.com/helyim/helyim?tab=readme-ov-file#benchmark)
